articles/errors/libraries/auth0-js/invalid-token.md

```diff
@@ -0,0 +1,23 @@
+---
+public: false
+---
+# HS256-Signed Tokens
+
+Error: calls made to the **/userInfo** return the following:
+
+```
+accessToken parameter is not valid
+```
+
+## Cause
```

Comment:
Not sure this section is needed, tbh. We already described the issue in the description above.